src/js/footer.js

```diff
@@ -0,0 +1,69 @@
+(function() { // protect the lemmings!
+
+    // in case we want to add stuff in the future
+    const GithubAPI = {};
+    GithubAPI.urlBase = `https://api.github.com`;
+    GithubAPI.endpoints = {
+        repos: `repos`,
+        users: `users`,
+    };
+    // super low sec access_token, just reads repos and public user data
+    // required to get over aggressive rate limiting of GH api
+    GithubAPI.access_token = '71b909c73044cce35b4f234762d81f4bca95a512';
```

Comment:
Hm where would this move to? Pulling it into BE for an additional call seems overkill imo